src/app/api/upload/route.ts

```diff
@@ -0,0 +1,98 @@
+import { md5 } from "@/lib/md5";
+import { apiError, concat } from "@/lib/utils";
+import withUser from "@/lib/with-user";
+import { NextResponse } from "next/server";
+import { ok, err } from "@justmiracle/result";
+import { R2 } from "@/lib/r2";
+import { db } from "@/db";
+import { user as userTable, user_file } from "@/db/schema";
+import { eq } from "drizzle-orm";
+import { generateId } from "lucia";
+
```

Comment:
Add

```
export const runtime = "edge";
```

This will make Vercel deploying it to the edge instead of fixed region

Reply:
Done